src/Paket.Core/ProjectFile.fs

```diff
@@ -639,7 +639,7 @@ type ProjectFile =
                         let c = input.[index]
                         if data.Length = 0 && c = ' '
                         then parseComparison data input (index + 1)
-                        elif data.Length = 2 && c <> ' '
+                        elif data.Length = 2 && c <> ' ' && c <> '\''
```

Comment:
What you wrote here should work fine for #1057.
If I had thought of this when I wrote the parser, I might have written this as something like this:

```FSharp
let isCompChar c = c = '<' || c = '>' || c = '!' || c = '='
...
elif data.Length = 2 && isCompChar c
```

(helper function so the next `elif` could be simplified as well)
If you want to do that with this PR, let me know how the tests go; otherwise this looks good to me :+1: